benchflow-dsl/src/main/scala/cloud/benchflow/dsl/definition/configuration/strategy/selection/SelectionStrategyType.scala

```diff
@@ -6,5 +6,9 @@ package cloud.benchflow.dsl.definition.configuration.strategy.selection
  */
 object SelectionStrategyType extends Enumeration {
   type SelectionStrategyType = Value
-  val Complete = Value("complete")
+  val OneAtATime = Value("one-at-a-time")
```

Comment:
As I remember we said that when there is a space we use `_` and when there is an actual `-` in the string we keep it. Normally you would write `One-at-a-Time` and not `One at a Time`